app/service/form_task_list_service.rb

```diff
@@ -107,8 +107,24 @@ def section_4_tasks
     [{
       task_name: I18n.t("forms.task_list_#{create_or_edit}.section_4.make_live"),
       path: @form.ready_for_live? ? make_live_path(@form.id) : "",
-      status: @task_list_statuses.make_live_status,
-      active: @task_list_statuses.mandatory_tasks_completed?,
+      status: @task_statuses[:make_live_status],
+      active: @form.ready_for_live?,
     }]
   end
+
+  def statuses_by_user
```

Comment:
This was necessary to remove any `User` concerns from task status service, so that it deals exclusively with form data and can be moved into forms-api